bin/next_ids

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+progdir=$(dirname "$0")
+printf "%-15s: %6d\n" \
+"Project" "$("$progdir/next_project_id")" \
```

Comment:
Looks fine how you have it, but _if_ you wanna make things a bit less noisy with quotes you could do something like

```
cd "$(dirname "$0")"
```

and then

```
Project  "$(./next_project_id)" \
...
```
etc.

(And really even the quotes around the `"$(...)"` are a bit pedantic given that we control the scripts, which are all careful to output exactly one positive integer token to stdout.)

Reply:
Sure